_includes/configuration/images-upload-handler.md

```diff
@@ -13,17 +20,21 @@ The upload handler function takes three arguments: `blobInfo`, a `success` callb
 ```js
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
-  images_upload_handler: function (blobInfo, success, failure) {
+  images_upload_handler: function (blobInfo, success, failure, progress) {
     var xhr, formData;
 
     xhr = new XMLHttpRequest();
     xhr.withCredentials = false;
     xhr.open('POST', 'postAcceptor.php');
 
+    xhr.upload.onprogress = function(e) {
```

Comment:
For consistency, maybe add a space after `function`?

Reply:
fixed by: https://github.com/tinymce/tinymce-docs/pull/1565/commits/2d0745a16dbd755d3bdf4cb745c053a8d06110b5